stable/logstash/values.yaml

```diff
@@ -75,7 +75,7 @@ ingress:
   #      - logstash.cluster.local
 
 # set java options like heap size
-logstashJavaOpts: "-Xmx1g -Xms1g"
```

Comment:
I don't feel like it's very transparent this way, users expect to see default values in `values.yaml` and make adjustments there according to their needs.

Reply:
@christian-roggia but as result, I see that the same options declared in JVM twice.
See image below
![image](https://user-images.githubusercontent.com/20466436/60808834-ef61ba80-a191-11e9-913e-c320ef2e43ef.png)